README.md

```diff
@@ -20,6 +20,32 @@ sequenceDiagram
 ```
 
 
+# Objectives and Explanation of OAuth Status Assertions
+
+Status assertions are used to provide time sensitive claims that impact the validity of digital credentials without the need for a direct request from third parties. They are periodically issued to the holders of the credentials, who can present them to the verifiers together with the corresponding digital credentials.
```

Comment:
"direct request from third parties" in the first sentence is a bit ambiguous.  Perhaps say instead: "without introducing third parties in the trust framework and changes in the flow of credential issuance and presentation." 

"They are periodically issued to the holders of the credentials" in the second sentence is not correct IMHO as they are issued per request of the Holder. So perhaps rephrase to: "They are issued on request of the credential Holder"

Reply:
Changes made, pls check